server.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+
+const routes = require('./api/routes/routes');
+
```

Comment:
I like the way you've separatd your package requirements from the file requirements 👍